src/tests/ftest/control/dmg_telemetry_io_basic.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2018-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from avocado.core.exceptions import TestFail
+
+from ior_test_base import IorTestBase
+from telemetry_test_base import TestWithTelemetry
+
+
+class TestWithTelemetryIOBasic(IorTestBase,TestWithTelemetry):
+    # pylint: disable=too-many-ancestors
+    """Test telemetry engine io basic metrics.
+
+    :avocado: recursive
+    """
+
+    def verify_io_test_metrics(self, io_test_metrics, metrics_data):
+        """ Verify telemetry io metrics from metrics_data.
```

Comment:
It doesn't look like this method is doing any type of verification. Just logging.

Reply:
Please see the comment I added.